torch/optim/swa_utils.py

```diff
@@ -362,16 +374,17 @@ def _get_initial_lr(lr, swa_lr, alpha):
 
     def get_lr(self):
         if not self._get_lr_called_within_step:
-            warnings.warn("To get the last learning rate computed by the scheduler, "
-                          "please use `get_last_lr()`.", UserWarning)
+            warnings.warn(
```

Comment:
how come there are so many spacing updates in this PR? would it be possible to keep the spacing the way it was?

Reply:
Fixed, this might cause by my wrong formatter config.